docs/contributing.md

```diff
@@ -120,6 +120,8 @@ There are other (intentional) differences between tests:
    only did, `dynamic` because they will become containers soon anyway to test cluster installs.)
 1. `dynamic` - Due to a limitation of Ansible 2.8 with collections, these are not tested with
    Ansible 2.8.
+1. The `release-upgrade` scenario uses its own `converge.yml` playbook instead of the default one
+   used by all other scenarios.
```

Comment:
This isn't so unique. What is unique is that the playbook does something extremely different. It runs repeated upgrades in one play.